### ISSUE TITLE
Rolling mill immersive animation

### ISSUE THREAD

**Issue by CKenJa** [mrh0/createaddition#83]
# problem
When I played this mod, I was excited.
However, I feel that I will be lacking the working animation of the rolling mill.
I suggest an immersive animation similar to the other Create components.

# solution(forced)
It's a very forced solution. Should research other methods.

## 1
Add an animation of items moving, like a mechanical saw, so that the output item drops.
It looks bad in 3D items. To solve this, we will display it in 2D like the inventory screen.

## 2
Make it look like the input items are rolling.
![image](https://user-images.githubusercontent.com/41111170/121802702-ad442c00-cc78-11eb-939e-40898795733d.png)
(iron ingot)

**Comment by mrh0** [comment]
Yes, I have thought of this but there are other things which as taken priority atm.
